Modules/Core/Common/include/itkBoundingBox.hxx

```diff
@@ -125,13 +125,6 @@ BoundingBox<TPointIdentifier, VPointDimension, TCoordRep, TPointsContainer>::Get
 }
 #endif
 
-/** */
-template <typename TPointIdentifier, unsigned int VPointDimension, typename TCoordRep, typename TPointsContainer>
-BoundingBox<TPointIdentifier, VPointDimension, TCoordRep, TPointsContainer>::BoundingBox()
-  : m_PointsContainer(nullptr)
-{
-  m_Bounds.Fill(NumericTraits<CoordRepType>::ZeroValue());
```

Comment:
Does `m_Bounds` still get initialized the same way?

Reply:
I think so, definitely. `m_Bounds` a FixedArray of `CoordRepType`, so it should be properly zero-initialized by its in-class `{}` initializer. Don't you agree?

Reply:
It might be un-initialized? Best to check in the debugger.

Reply:
> It might be un-initialized? Best to check in the debugger.

Just submitted pull request #3881, to show that an empty `{}` properly initializes a FixedArray. Does that help?